### DISCUSSION TITLE
Displaying values for rotary controls

### DISCUSSION THREAD

**Discussion by iankbr** [calibx/msfsdeck#67] [Q&A]
I'm seriously considering getting a LoupeDeck to replace my Stream Deck XL and my set of cumbersome Logitech Saitek Panels.

However, the Saitek Autopilot Panel has the advantage of having a display of the current value updated by the rotary encoder.

With your LoupeDeck profile, will the description panel corresponding to a rotary encoder be able to display the dynamic value selected by that encoder.  And, if not, can a neighbouring button display that dynamic value.

Ian.

**Comment by calibx** [comment]
Hi,

You can do all you want for the encoders with some lines of C# 😄 

In the plugin, I display 2 values for each AP encoder (AP value between [ ] and actual value). I've made a little video to explain it :https://vimeo.com/680796523

Seb

**Reply by iankbr** [reply]
Great!

C# is my main language, so even better.

My Loupedeck should be delivered today, so hopefully I will be playing with it tonight.

Keep up the great work,
Ian.
